src/responsive-layout.tsx

```diff
@@ -64,16 +65,20 @@ const ResponsiveLayout: React.FC<Props> = ({children, className, fullWidth, data
 
     const sideMargin = isMobile ? MOBILE_SIDE_MARGIN : isTablet ? TABLET_SIDE_MARGIN : null;
 
+    const containerType = isMobile ? 'mobile-column' : isTablet ? 'tablet-column' : 'desktop-wide-column';
+
     return (
         <ResponsiveLayoutMarginContext.Provider value={sideMargin}>
-            <div
-                className={classnames(classes.container, className)}
-                {...getPrefixedDataAttributes(dataAttributes)}
-            >
-                <div className={classnames(classes.layout, {[classes.layoutFullWidth]: fullWidth})}>
-                    {children}
+            <ContainerTypeContext.Provider value={containerType}>
```

Comment:
I wouldn't apply this to responsive layout, only to grid layout

Reply:
Appliyng to responsive layout we can also enable the components to use the Context on all devices, simplifying the development of the components (they can then take into account only the container type).